src/internal/components/pluggableVisualizations/comboChart/tests/PluggableComboChart.spec.tsx

```diff
@@ -470,4 +472,63 @@ describe("PluggableComboChart", () => {
             ]);
         });
     });
+
+    describe("default chart type", () => {
+        it("should return default chart type", async () => {
+            const refPointMock = referencePointMocks.multipleMetricBucketsAndCategoryReferencePoint;
+            const extRefPoint: IExtendedReferencePoint = await getExtendedReferencePoint(refPointMock);
+            const controls = get(extRefPoint, PROPERTY_CONTROLS, {});
+
+            expect(controls.primaryChartType).toBe("column");
+            expect(controls.secondaryChartType).toBe("line");
```

Comment:
please use the existing constants

Reply:
In case we use contants to assert, the test would pass in case value of constant is updated, and we should be aware of any changes would make test failed.